torch/_dynamo/variables/builder.py

```diff
@@ -2991,12 +2991,8 @@ def handle_traced_output(example_value, tx, proxy, options, subclass_type, targe
         return ConstantVariable.create(example_value, **options)
     elif isinstance(example_value, (int, float, bool)) and (
         proxy.node.target is call_torchbind
-    ):
-        set_example_value(proxy.node, example_value)
-        return ConstantVariable.create(example_value, **options)
-    elif (
-        isinstance(example_value, (int, float, bool))
-        and proxy.node.target is flat_apply
+        or proxy.node.target is flat_apply
+        or (proxy.node.op == "call_method" and proxy.node.target == "item")
```

Comment:
Is this the preferred way to check for `Tensor.item()`? I don't know if other cases could undesirably go through this code path.

Reply:
I think soo? like [this](https://github.com/pytorch/pytorch/blob/1ec2c15914da4ef7bd926ed9aebc8671c75fe965/torch/_dynamo/output_graph.py#L2340)